app/components/app-avatar-image-fallback.component.scss

```diff
@@ -0,0 +1,23 @@
+.fallback {
+    background: #d6dbe7;
+    border: 5px solid #fff;
+    border-radius: 50%;
+    height: 72px;
+    min-height: 72px;
+    min-width: 72px;
+    object-fit: cover;
+    width: 72px;
+}
+
+.app-avatar-image {
+    line-height: 2em;
+    text-align: center;
+    height: 62px;
+    width: 62px;
+    border-radius: 50%;
+    font-size: 39px;
+    display: flex !important;
+    justify-content: center;
+    align-items: center;
+    color: #FFFFFF;
+}
```

Comment:
lets give these BEM style classnames so they don't have any potential conflicts. I could especially see "fallback" being a classname in another app at some point.